src/java/com/jalasoft/search/model/Search.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018 Jalasoft.
+ *
+ * This software is the confidential and propietary information of Jalasoft.
+ */
+package java.com.jalasoft.search.model;
+/**
+ * <Class description goes here>
+ *
+ * @author sofia cespedes
+ * @version 1.0
+ */
+import java.io.File;
+
+public class Search {
+    public void Search() {
+
+    }
+
+    public void ListFileByPath(String name, File file) {
+        File[] listOfFiles = file.listFiles();
```

Comment:
Please add Documentation related to the method. 
e.g:
    /**
     * Verifies if the path is valid
     *
     * @return boolean
     */